documentation/feed-using-hadoop-pig-oozie.html

```diff
@@ -448,38 +448,32 @@ <h3 id="efficiency">Efficiency</h3>
 a number of mappers or reducers will be used to cover the file if the input file is sufficiently large.
 As each mapper will have their own client feeding to the endpoint, each with multiple threads,
 make sure the cluster can take the load.
-Each client will throttle to not overload the endpoint.
-Using a high number of effective thread (tasks times threads per mapper),
-can easily saturate feeding endpoints.
+Each client will throttle to not overload the endpoint but clients does not know how many total client there are so one can easily over saturate 
```

Comment:
Each client will throttle to not overload the endpoint**,** but **the** clients do not know how many total client**s** there are, so one can easily over**-**saturate *the* feeding endpoints.